src/timeline/js/controllers.js

```diff
@@ -425,12 +421,12 @@ App.controller('TimelineCtrl',function($scope) {
  // Set the audio data for a clip
  $scope.setAudioData = function(clip_id, audio_data) {
  	// Find matching clip
-	for (var clip_index = 0; clip_index < $scope.project.clips.length; clip_index++) {
-		if ($scope.project.clips[clip_index].id == clip_id) {
+	for (const clip of $scope.project.clips) {
+		if (clip.id == clip_id) {
 			// Set audio data
 			$scope.$apply(function(){
-				$scope.project.clips[clip_index].audio_data = audio_data;
-				$scope.project.clips[clip_index].show_audio = true;
+				clip.audio_data = audio_data;
+				clip.show_audio = true;
 			});
```

Comment:
I mentioned that `for...of` often makes the code cleaner: This change makes for a good representative example. The updated version is _so_ much more readable.

Reply:
:+1: